test/src/main/java/org/apache/accumulo/test/functional/BulkNewIT.java

```diff
@@ -183,6 +184,23 @@ public void testSingleTabletSingleFileOffline() throws Exception {
     }
   }
 
+  @Test
+  public void testMaxTablets() throws Exception {
+    String maxTablets = "0";
+    try (AccumuloClient client = Accumulo.newClient().from(getClientProps()).build()) {
+      maxTablets = client.instanceOperations().getSystemConfiguration()
+          .get(Property.MASTER_BULK_MAX_TABLETS.getKey());
+      client.instanceOperations().setProperty(Property.MASTER_BULK_MAX_TABLETS.getKey(), "1");
```

Comment:
I am not completely sure, but these two lines may not work as expected sometimes.  What I think may be able to happen is that the property is set on tserver A by the first line and then the second line reads props from tserver B.  If server B does not clear zoocache, then it could return a stale copy w/o the prop set on the first line.  I am not sure if the client side code to get props forces a prop cache clear.

Reply:
If it were a table prop, then table could be created with the prop and would not need to clear in finally.